week-1/Homework/mandatory/1-syntax-errors.js

```diff
@@ -1,14 +1,15 @@
 // There are syntax errors in this code - can you fix it to pass the tests?
 
-function addNumbers(a b c) {
+function addNumbers(a, b, c) {
     return a + b + c;
 }
 
-function introduceMe(name, age)
-return "Hello, my name is " + name "and I am " age + "years old";
-
+function introduceMe(name, age){
+return "Hello, my name is " + name + "and I am " + age + "years old";
```

Comment:
Good job! Have you considered using string interpolation here instead to make your code neater? What about indentation here?